src/contexts/crudOperations-context.js

```diff
@@ -8,7 +8,6 @@ const CrudOperationsProvider = ({ children }) => {
     const notesReducer = ( stateOfNotes, action ) => {
```

Comment:
nice you have created context for CURD operation

Reply:
> short and clean code , i think your join now link is not working in preview check ones more

yeah!! will work on that part!! haven't made the ui for signup page yet!! or may be will change the link by directing it to homepage instead of signup!!